ocs_ci/ocs/bucket_utils.py

```diff
@@ -1424,3 +1424,66 @@ def write_random_test_objects_to_bucket(
         signed_request_creds=s3_creds,
     )
     return obj_lst
+
+
+def write_data_to_buckets_and_check_integrity(
```

Comment:
Is this use case common enough to warrant a function of its own?
I think it'd probably be better to turn this into the test's logic.